components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1154,58 +1156,86 @@ void AdsServiceImpl::URLRequest(
       const std::string& content_type,
       ads::URLRequestMethod method,
       ads::URLRequestCallback callback) {
-  net::URLFetcher::RequestType request_type = URLMethodToRequestType(method);
-
-  net::URLFetcher* fetcher = net::URLFetcher::Create(
-      GURL(url), request_type, this).release();
-  fetcher->SetRequestContext(g_browser_process->system_request_context());
-  fetcher->SetAutomaticallyRetryOnNetworkChanges(kRetriesCountOnNetworkChange);
-
+  net::NetworkTrafficAnnotationTag traffic_annotation =
+      net::DefineNetworkTrafficAnnotation("ads_service_impl", R"(
+          semantics {
+            sender: "Brave Ads service"
+            description: "..."
+            trigger: "..."
+            data: "..."
+            destination: WEBSITE/GOOGLE_OWNED_SERVICE/OTHER
+          }
+          policy {
+            cookies_allowed: NO/YES
+            cookies_store: "..."
+            setting: "..."
+            chrome_policy {
+              [POLICY_NAME] {
+                  [POLICY_NAME]: ...
+              }
+            }
+            policy_exception_justification = "..."
+          }
+          comments: "..."
+        )");
+
+  auto request = std::make_unique<network::ResourceRequest>();
```

Comment:
probably we want
```
resource_request->load_flags =
      net::LOAD_DO_NOT_SEND_COOKIES | net::LOAD_DO_NOT_SAVE_COOKIES;
```

Reply:
Thanks for checking!
`request->allow_credentials = false;` is added here.
`DO_NOT_SAVE_COOKIES` and related flags are deprecated.